src/main/java/de/retest/ui/descriptors/Element.java

```diff
@@ -58,28 +58,8 @@ protected Element() {
 		containedComponents = new ArrayList<>();
 	}
 
-	public Element( final String retestId, final IdentifyingAttributes identifyingAttributes,
-			final Attributes attributes ) {
-		this( retestId, identifyingAttributes, attributes, new ArrayList<Element>() );
-	}
-
-	public Element( final String retestId, final IdentifyingAttributes identifyingAttributes,
-			final Attributes attributes, final Element... containedComponents ) {
-		this( retestId, identifyingAttributes, attributes, new ArrayList<>( Arrays.asList( containedComponents ) ) );
-	}
-
 	public Element( final String retestId, final IdentifyingAttributes identifyingAttributes,
```

Comment:
If we want to use factory methods, the ctor should be `private`. We shouldn't allow both.